src/cms/locale/de/LC_MESSAGES/django.po

```diff
@@ -365,10 +369,15 @@ msgstr ""
 msgid "The end of the event can't be before the start of the event"
 msgstr "Das Ende der Veranstaltung kann nicht vor seinem Beginn sein"
 
+<<<<<<< HEAD
+#: forms/events/event_translation_form.py:100
+#: forms/pois/poi_translation_form.py:103
+=======
 #: forms/events/event_translation_form.py:135
 #: forms/imprint/imprint_translation_form.py:122
 #: forms/pages/page_translation_form.py:140
 #: forms/pois/poi_translation_form.py:149
+>>>>>>> 2682b9718108d6151c2b25194ff78b399038cc8a
```

Comment:
The sections marked with `<<<<<<< HEAD`, `=======` and `>>>>>>>` denote [git merge conflicts](https://www.atlassian.com/git/tutorials/using-branches/merge-conflicts). Please always resolve conflicts before committing changes.
For the translation file, we luckily have the dev tool `./dev-tools/resolve_translation_conflicts.sh` which can resolve these conflicts automatically.